nikola/plugins/compile/ipynb/__init__.py

```diff
@@ -31,8 +31,15 @@
 import os
 
 try:
+    import IPython
     from IPython.nbconvert.exporters import HTMLExporter
-    from IPython.nbformat import current as nbformat
+    if IPython.version_info[0] >= 3:     # API changed with 3.0.0
+        from IPython import nbformat
+        current_nbformat = nbformat.current_nbformat
+    else:
+        import IPython.nbformat.current as nbformat
+        current_nbformat = 'json'
```

Comment:
Maybe this should be a Unicode string (``u'json'``)?  (I don't think it matters though, but have not really done much with unicode strings yet...)